.github/workflows/release-snapshot.yml

```diff
@@ -0,0 +1,29 @@
+name: Release Snapshot to Sonatype
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  release_snapshot_sonatype:
+    if: "github.event.release.prerelease"
```

Comment:
In the github UI, when creating a snapshot release, we'll expect that the release to be marked as one
<img width="1017" alt="Screenshot 2023-01-20 at 15 04 39" src="https://user-images.githubusercontent.com/45561419/213730939-a9bc6cc6-043a-4ffd-a2f2-39615c02f314.png">

Reply:
👍